testdata/p4_16_samples_outputs/default-switch.p4-stderr

```diff
@@ -1,4 +1,4 @@
-default-switch.p4(13): warning: SwitchCase: label following default label
+default-switch.p4(13): [--Wwarn=ordering] warning: SwitchCase: SwitchCase: label following default label
```

Comment:
there are a whole bunch of errors from now on where there is a duplicated IR node name.

Reply:
fixed.